### PR TITLE
small space ruin

### PR DESCRIPTION
**Pull request by Yobrocharlie**
Smol space ruin
## About The Pull Request
Adds a small space ruin to space, a small vault with a safe
smallspaceruin

## Why It's Good For The Game
Adds more content to space
## Changelog
adds a very small vault to space,
with an empty filing cabinet, a vault with a very small amount of money a russian revolver syndicate files and an empty bag of money.
:cl:
add: Added more things
/:cl:

### CODE REVIEW

File: _maps/RandomRuins/SpaceRuins/LandmineVault.dmm [hunk 1]
```diff
@@ -0,0 +1,391 @@
...
+/obj/effect/decal/cleanable/dirt/dust,
+/turf/open/floor/plating/airless,
+/area/ruin/space)
+"m" = (
+/obj/effect/decal/cleanable/dirt,
+/turf/open/floor/plating/airless,
+/area/ruin/space)
+"p" = (
+/obj/structure/safe,
+/obj/item/book/granter/crafting_recipe/dusting/laser_musket_prime,
```
Comment (necromanceranne) [inline _maps/RandomRuins/SpaceRuins/LandmineVault.dmm:33]: I believe it has been said on numerous occasions that mapping in or making it easier/more reliable to obtain something rare or difficult to acquire is not a good thing, and this isn't an exception. Please don't map in the prime crafting books.

Comment (Yobrocharlie) [reply _maps/RandomRuins/SpaceRuins/LandmineVault.dmm:33]: Sure

**Comment by Pumpkinoe** [comment]
i feel like the Why It's Good For The Game section should be a little more than just 'adds more content' for something like space ruins

what's the point of this space ruin other than just giving exploding someone who hasn't seen it before? what's the theme here? what's the purpose of the vault door? do you have to hack in?

you have also, once again, left your changelog as just 'Added more things' without anything to describe what you've added.

**Comment by DaydreamIQ** [comment]
A single ruin that's just a small box with documents in it really doesn't seem like its useful for anything except giving exploring antags free access to those things. This also feels pretty devoid of any character whatsoever, just a boring 5x5 box with nothing else attached

**Comment by Pumpkinoe** [comment]
> A single ruin that's just a small box with documents in it really doesn't seem like its useful for anything except giving exploring antags free access to those things. This also feels pretty devoid of any character whatsoever, just a boring 5x5 box with nothing else attached

Don't forget the explosive mine that's hidden by the nuke toy! The one presumably designed to blow up people who know nothing about the ruin!

**Comment by ShizCalev** [comment]
image

**Pull request closed without merging**